draft-ietf-tls-esni.md

```diff
@@ -416,9 +416,21 @@ OuterExtensions consists of one or more ExtensionType values, each of which
 reference an extension in ClientHelloOuter.
 
 When sending ClientHello, the client first computes ClientHelloInner, including
-any PSK binders. It then computes a new value, the EncodedClientHelloInner, by
-first making a copy of ClientHelloInner. It then replaces the
-legacy\_session\_id field with an empty string.
+any PSK binders. It then computes a new value, the EncodedClientHelloInner,
+which is the following structure:
+
+~~~
+    struct {
+        ClientHello client_hello;
+        opaque padding[N];
```

Comment:
nit: `uint8 zeros[length_of_padding];` aligns a little nicer with [8446](https://datatracker.ietf.org/doc/html/rfc8446#section-5.2).

Reply:
Done. It does make it no longer align with the requested variable name change in {{padding}}, but I agree with you aligning with 8446 makes sense.